tests/analytics_expected_loaded_fields.txt

```diff
@@ -0,0 +1,11 @@
+*_rel_iris
```

Comment:
I have the impression from code above that this file is being generated all the time. If that is the case, maybe remove from version control.